src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ name, email, body }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{name}</strong>
+
+      {' by '}
+
+      <a
+        className="CommentInfo__email"
+        href={`mailto:${email}`}
+      >
+        {email}
+      </a>
+    </div>
+
+    <div className="CommentInfo__body">
+      {body}
+    </div>
+  </div>
 );
```

Comment:
```suggestion
export const CommentInfo = ({ comment }) => {
const { name, email, body } = comment;

return (
  <div className="CommentInfo">
    <div className="CommentInfo__title">
      <strong className="CommentInfo__name">{name}</strong>

      {' by '}

      <a
        className="CommentInfo__email"
        href={`mailto:${email}`}
      >
        {email}
      </a>
    </div>

    <div className="CommentInfo__body">
      {body}
    </div>
  </div>
);
}
```

Add restructuring like this.